iai_semantic_maps/owl/kitchen.owl

```diff
@@ -917,6 +917,16 @@
         <knowrob:hasVisual rdf:datatype="&xsd;boolean">false</knowrob:hasVisual>
     </owl:NamedIndividual>
 
+    <owl:NamedIndividual rdf:about="&iai-map;iai_kitchen_meal_table_counter_top">
+        <rdf:type rdf:resource="&knowrob;CounterTop"/>
+        <knowrob:depthOfObject rdf:datatype="&xsd;double">2.45</knowrob:depthOfObject>
+        <knowrob:heightOfObject rdf:datatype="&xsd;double">0.02</knowrob:heightOfObject>
+        <knowrob:widthOfObject rdf:datatype="&xsd;double">0.8</knowrob:widthOfObject>
```

Comment:
Nobody told me those; can fix later.

> On 01.08.2016, at 15:08, Gayane Kazhoyan <notifications@github.com> wrote:
> 
> In iai_semantic_maps/owl/kitchen.owl:
> 
> > @@ -917,6 +917,16 @@
> >          <knowrob:hasVisual rdf:datatype="&xsd;boolean">false</knowrob:hasVisual>
> >      </owl:NamedIndividual>
> >  
> > +    <owl:NamedIndividual rdf:about="&iai-map;iai_kitchen_meal_table_counter_top">
> > +        <rdf:type rdf:resource="&knowrob;CounterTop"/>
> > +        <knowrob:depthOfObject rdf:datatype="&xsd;double">2.45</knowrob:depthOfObject>
> > +        <knowrob:heightOfObject rdf:datatype="&xsd;double">0.02</knowrob:heightOfObject>
> > +        <knowrob:widthOfObject rdf:datatype="&xsd;double">0.8</knowrob:widthOfObject>
> The dimensions are 90 x 160 x 76.
> Could you please use those?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>